tests/src/io.jsonwebtoken/jjwt-jackson/0.11.5/user-code-filter.json

```diff
@@ -0,0 +1,13 @@
+{
+  "rules": [
+    {
+      "excludeClasses": "**"
+    },
+    {
+      "includeClasses": "io.jsonwebtoken.jackson.**"
+    },
+    {
+      "includeClasses": "io.jsonwebtoken.**"
+    }
+  ]
+}
```

Comment:
- In fact, `io.jsonwebtoken:jjwt-jackson`, `io.jsonwebtoken:jjwt-orgjson` and `io.jsonwebtoken:jjwt-gson`, they all require `io.jsonwebtoken:jjwt-api` and `io.jsonwebtoken:jjwt-impl`s full or partial GraalVM reachability metadata. 
- I wonder if there is any easier way to directly import the GraalVM reachability metadata of `io.jsonwebtoken:jjwt-impl` to guarantee `./gradlew clean test -Pcoordinates=io.jsonwebtoken:jjwt-jackson:0.11.5 `, won't the execution fail?

Reply:
- I don't think GraalVM reachability metadata for `io.jsonwebtoken:jjwt-impl:0.11.5` should be independent in this case.